dot/core/messages.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/ChainSafe/gossamer/dot/network"
 	"github.com/ChainSafe/gossamer/dot/peerset"
 	"github.com/ChainSafe/gossamer/dot/types"
-	"github.com/ChainSafe/gossamer/lib/runtime"
+	runtimeErrors "github.com/ChainSafe/gossamer/lib/runtime/errors"
```

Comment:
supernit convention seems to be package names should be all lowercase, so rename to `runtimererrors`